File: Lista2/C++/Szesciokat.cpp
#include <iostream>
#include <cmath>
#include "Szesciokat.h"

Szesciokat::Szesciokat(double bok)
{
    this->bok = bok;
}

double Szesciokat::obwod()
{
    return (6 * bok);
}

double Szesciokat::pole()
{
    return (3 * sqrt(3) * pow(bok, 2)/ 2 );
}

Szesciokat::~Szesciokat()
{
	
}

File: Lista2/Java/Szesciokat.java
public class Szesciokat extends Figura
{
    private double bok;

    Szesciokat(double bok)
    {
        this.bok = bok;
    }

    public double obwod()
    {
        return (6 * bok);
    }

    public double pole()
    {
        return (3 * Math.pow(bok, 2) * Math.sqrt(3) / 2);
    }
}